### PR TITLE
begin work on enabling oscap-ssh script functionality

### PR DESCRIPTION
**Pull request by openprivacy**
Not really ready for merging, but I think this is a step forward in that it matches the existing behavior - though with better (IMO) results file naming - and has some initial support for remote (oscap-ssh) scanning.

If you test this and it works for local scanning as expected, perhaps a merge would be good so we both have the same starting place.

### CODE REVIEW

**Comment by openprivacy** [comment]
I believe this pull request is ready to rock. It supports remote scanning (for a single remote host configured in GovReadyfile) and fix. Still to do when using datastream for remote host:
- Quickie report not generated
- `govready compare` not working
- OVAL variables are not created

The first two items above are rather important - do you have time to resolve?

**Comment by openprivacy** [comment]
Added functionality to enable Ansible to set GOVREADY_* environment variables so that sites in the inventory can be scanned. Not yet working with data streams:
- quickie report
- govready compare

File: govready [hunk 1]
```diff
@@ -45,19 +45,19 @@ __log_govready() {
...
-SUFFIX="$(date +%m%d-%H%M)"
+# Time at which scanning is initiated
+DATE_TIME="$(date +%Y%m%d-%H%M)"
 
 # Set location of xsl files
 FILTERRESULTSFILE=".govready/xml/filterresults.xsl"
 RULEIDRESULTFILE=".govready/xml/ruleidresult.xsl"
 
-
+CONFIGFILE="GovReadyfile"
```
Comment (gregelin) [inline govready]: +1

**Comment by gregelin** [comment]
Thanks for awesome improvements!

**Pull request merged**